R/get_sample_cn_segments.R

```diff
@@ -2,16 +2,17 @@
 #'
 #' @description Get all segments for a single (or multiple) sample_id(s).
 #'
-#' @details This function returns CN segments for samples. This works for single sample or multiple samples.
+#' @details This function returns CN segments. This works for single sample or multiple samples.
 #' Specify the sample IDs you are interested in with `these_sample_ids` (as a vector of characters),
-#' Or call this function with `these_samples_metadata` if you already ahve a metadata table subset to the sample IDs of interest.
-#' If none off the above parameters are specified, the function will return CN segments for available samples.
+#' Or call this function with `these_samples_metadata` if you already have a metadata table subset to the sample IDs of interest.
+#' If none of the above parameters are specified, the function will return CN segments for available samples (from get_gambl_metadata).
 #' Note, this. function internally calls [GAMBLR.data::id_ease] for dealing with sample IDs and metadata tables. 
 #'
-#' @param these_sample_ids Optional argument, sample_id (vector of characters) for the sample(s) to retrieve segments for. If not provided, the function will return CN segments for all available sample IDs present in the current metadata.
-#' @param these_samples_metadata Optional, provide a metadata (data frame) subset to the sample IDs of interest.
+#' @param these_sample_ids Optional, a vector of multiple sample_id (or a single sample ID as a string) that you want results for.
```

Comment:
The `GAMBLR` and `GAMBLR.results` versions of `get_sample_cn_segments` use the parameter `this_sample_id` instead of `these_sample_ids`. We probably want to keep `this_sample_id` to not change published code, but the plural form might make more sense.

Reply:
I agree the plural makes more sense here, and we should keep it here as suggested in the PR. So to harmonize, would need to update the naming of this parameter in the `GAMBLR.results`